sonarqube-ant-task/pom.xml

```diff
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>org.sonarsource.scanner.api</groupId>
       <artifactId>sonar-scanner-api</artifactId>
-      <version>2.8</version>
+      <version>2.9.0.887</version>
```

Comment:
Should directly update to the latest version, that is 2.12.0.1661. But I will take care of it, thanks: https://jira.sonarsource.com/browse/ANTTASK-58